app/graphql/types/reservation_type.rb

```diff
@@ -0,0 +1,13 @@
+module Types
+  class ReservationType < Types::BaseObject
+    graphql_name 'Rezervation'
+    description 'Rezerwacje'
```

Comment:
Może trzymajmy się angielskiego w opisach też

Reply:
Ok to pozmieniam w poprzednich rzeczach też.